src/OrbitCaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -159,6 +159,13 @@ std::vector<TimerInfo> GpuQueueSubmissionProcessor::ProcessGpuQueueSubmissionWit
   std::optional<GpuCommandBuffer> first_command_buffer =
       ExtractFirstCommandBuffer(gpu_queue_submission);
 
+  // The first command buffer acts as our reference needed to align GPU time based events in the
+  // CPU timeline. If we have no result for it -- which is the case if we started capturing within
```

Comment:
Maybe change "result" to "initial GPU timestamp" (if I got it correctly)?

Reply:
Somewhat. I changed it to "first timestamp of the submission"